src/errorHandler.js

```diff
@@ -0,0 +1,39 @@
+const availableCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+const VALIDATION_ERRORS = {
+  NO_TEXT:
+    'Text to convert is required. Correct request is: ' +
+    '"/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+  NO_TARGET_CASE:
+    '"toCase" query param is required. Correct request is: ' +
+    '"/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+  CASE_NOT_SUPPORTED:
+    'This case is not supported. Available cases: ' +
+    `${availableCases.join(', ')}.`,
+};
+
+function errorHandler(originalText, targetCase) {
+  const errors = [];
+
+  if (!originalText) {
+    errors.push({
+      message: VALIDATION_ERRORS.NO_TEXT,
+    });
+  }
+
+  if (!availableCases.includes(targetCase) && targetCase) {
+    errors.push({
+      message: VALIDATION_ERRORS.CASE_NOT_SUPPORTED,
+    });
+  }
+
+  if (!targetCase) {
```

Comment:
The check for `!targetCase` should be performed before checking if the case is unsupported. This ensures that the error for a missing target case is prioritized over an unsupported case.